src/script/lua_api/l_mainmenu.cpp

```diff
@@ -99,7 +99,7 @@ int ModApiMainMenu::l_update_formspec(lua_State *L)
 	//read formspec
 	std::string formspec(luaL_checkstring(L, 1));
 
-	if (engine->m_formspecgui) {
+	if (engine->m_formspecgui != 0) {
```

Comment:
?

Reply:
I just undid the diff. The file is no longer touched by the PR.